src/main/webapp/dev/js/common/questionRubric.es6

```diff
@@ -10,6 +10,110 @@ import {
     disallowNonNumericEntries,
 } from './ui.es6';
 
+function getRubricChoiceElem(questionNum, col) {
+    return $(`#rubricChoice-${questionNum}-${col}`);
+}
+
+function getRubricWeightElem(questionNum, col) {
+    return $(`#rubricWeight-${questionNum}-${col}`);
+}
+
+function getRubricDescElem(questionNum, row, col) {
+    return $(`#rubricDesc-${questionNum}-${row}-${col}`);
+}
+
+function swapRubricCol(questionNum, colIndex, isSwapLeft) {
```

Comment:
I refactored code a bit. You might find the new version more pleasing. :sweat_smile: